src/internal.c

```diff
@@ -1946,6 +1946,11 @@ void InitSuitesHashSigAlgo(Suites* suites, int haveECDSAsig, int haveRSAsig,
     #ifndef NO_SHA256
         AddSuiteHashSigAlgo(suites, sha256_mac, rsa_sa_algo, keySz, &idx);
     #endif
+#ifdef OPENSSL_ALL
```

Comment:
I don't see how this applies to our OpenSSL compatibility layer.

OPENSSL_EXTRA should be used for OpenSSL compatibility APIs that are not in default build and for behavior that is unique to OpenSSL which conflicts with wolfSSL.